pkg/apis/servicecatalog/types.go

```diff
@@ -297,4 +297,10 @@ const (
 	BindingConditionReady BindingConditionType = "Ready"
 	// BindingConditionFailed represents a binding condition is in failed state
 	BindingConditionFailed BindingConditionType = "Failed"
+	// BindingConditionUninject represents a binding condition that the binding credentials have been deleted
+	BindingConditionUninject BindingConditionType = "Uninject"
```

Comment:
I don't think it matters what tense they are in as long as we are clear about the order of the condition list and the description of what the conditions are.